sql-plugin/src/main/scala/com/nvidia/spark/rapids/Plugin.scala

```diff
@@ -374,13 +374,12 @@ class RapidsExecutorPlugin extends ExecutorPlugin with Logging {
         case Some(value) => ZoneId.of(value)
         case None => throw new RuntimeException(s"Driver time zone cannot be determined.")
       }
-      if (TypeChecks.areTimestampsSupported(driverTimezone)) {
```

Comment:
may off-topic. Considering the configuration ```spark.sql.session.timeZone```, should both driver and executor respect it? Then do we still need the check on driver and executor's timezone mismatch?

Reply:
> Considering the configuration spark.sql.session.timeZone, should both driver and executor respect it?

Here `driverTimezone` is from driver `ZoneId.systemDefault()`, not from `spark.sql.session.timeZone`, refer to: [PR](https://github.com/NVIDIA/spark-rapids/pull/4129)
Spark itself does not have this kind of check.

But for our `spark-rapids`, we check executor and driver have the same JVM time zone.